apps/rush-lib/src/logic/policy/PolicyValidator.ts

```diff
@@ -3,13 +3,22 @@
 
 import { RushConfiguration } from '../../api/RushConfiguration';
 import { GitEmailPolicy } from './GitEmailPolicy';
+import { LockfilePolicy } from './LockfilePolicy';
+
+export interface IPolicyValidatorOptions {
+  bypassPolicy?: boolean,
+  allowShrinkwrapUpdates?: boolean,
+  shrinkwrapVariant?: string
+}
 
 export class PolicyValidator {
-  public static validatePolicy(rushConfiguration: RushConfiguration, bypassPolicy: boolean): void {
-    if (bypassPolicy) {
+  public static validatePolicy(
+    rushConfiguration: RushConfiguration, options: IPolicyValidatorOptions): void {
```

Comment:
Why not just use `IInstallManagerOptions`?

Reply:
It has a bunch of properties that need to be populated because they aren't optional. There's some other code calling into this directly that doesn't come from the InstallManager (like PublishAction and VersionAction).

Given this, I figured that giving the options their own interface and making everything optional made the most sense.